Music Advisor/task/src/advisor/controller/Controller.java

```diff
@@ -0,0 +1,75 @@
+package advisor.controller;
+
+import advisor.controller.commands.Command;
+import advisor.controller.commands.CommandFactory;
+import advisor.model.DataSource;
+import advisor.view.View;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.List;
+
+public class Controller {
+    private View view;
+    private DataSource source = new DataSource();
+
+    public Controller(View view) {
+        this.view = view;
+    }
+
+    public void start(String access, String resource) throws IOException {
```

Comment:
it will be a good idea to make this method smaller by extracting some lines of code to separate methods